lnrpc/lightning.proto

```diff
@@ -1531,6 +1531,9 @@ message Channel {
 
     // This is the confirmed / on-chain zero-conf SCID.
     uint64 zero_conf_confirmed_scid = 33;
+
+    // This is the peer SCID alias.
+    uint64 peer_scid_alias = 34 [jstype = JS_STRING];
```

Comment:
I suggest moving this to be next to `repeated uint64 alias_scids = 31;` as in my understanding this would put it next to it for the API docs.

Reply:
Needs to be updated to field number 35!